docs/devsetup/java/maven/index.md

```diff
@@ -106,7 +106,22 @@ work as they do normally in IntelliJ.
 
 [Eclipse's Maven plugin](https://eclipse.dev/m2e/)
 works reasonably well and can support the import of a multi-module Maven project.
-These instructions have not yet verified Eclipse's handling of the import of the ICU4J using a Maven build.
+
+1. It is highly recommended (but optional) to do the ICU work in a separate workspace. \
+   You can also import to a working-set. Whatever you prefer. \
+   A workspace allows you to configure the Eclipse options for ICU at workspace level,
+   not at module level. It also separates the (many) ICU modules from the other work
+   you do. The cons is that every time you open Eclipse you will have to select
+   the workspace to work on.
+   See https://www.ibm.com/docs/en/baw/23.0.x?topic=modules-workspaces
```

Comment:
Markdown formatting specific to our docs theme & doc site toool: plain links need to be either

* surrounded by angle brackets (`<...>`)
* turned into a typical link that linkifies text `[..](...)`

If not, then they are unlinkified plain text, which is ugly and gets overlooked.